packages/components/src/RichTooltip/RichTooltip.component.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Actions } from '../Actions';
+import CircularProgress from '../CircularProgress';
+import Icon from '../Icon';
+import Inject from '../Inject';
+import theme from './RichTooltip.scss';
+
+export function getContent(props) {
+	let content;
+
+	if (typeof props.Content === 'string') {
+		content = props.Content;
+	}
+
+	if (props.loading) {
+		content = (
+			<div className={theme['tc-center-content']}>
+				<CircularProgress />
+			</div>
+		);
+	}
+
+	if (props.error) {
+		content = (
+			<div className={classNames(theme['tc-center-content'], theme['tc-error-wrapper'])}>
+				<Icon name="talend-warning" className={theme['tc-center-icon']} />
+				<div className={theme['tc-error-body']}>
+					<h1>Whoops !</h1>
+					<div>{props.error}</div>
+				</div>
+			</div>
+		);
+	}
+
+	return content;
```

Comment:
I prefer to use else here the priority is reversed, so error is the most important.
if (props.error){}
else if (props.loading) {}
else if (typeof === string) {}

with return in it will make it more clear